opennav_coverage/CMakeLists.txt

```diff
@@ -42,6 +43,7 @@ set(dependencies
   builtin_interfaces
   tf2_ros
   opennav_coverage_msgs
+  ament_index_cpp
```

Comment:
No idea why, but only in this package do it get:

```
/opt/overlay_ws/src/opennav_coverage/opennav_coverage/test/test_utils.cpp:21:10: fatal error: ament_index_cpp/get_package_share_directory.hpp: No such file or directory
   21 | #include "ament_index_cpp/get_package_share_directory.hpp"
```

although other packages have this include and don't explicitly have ament_index_cpp in CMakelist

Reply:
That is really odd. I didn't see that but maybe I need to break my own workspace cache to have seen that.

Reply:
I'd say just to leave it - it doesn't hurt anyone.